### PR TITLE
make `make gosec` output slightly easier to find

### PR DESCRIPTION
**Pull request by dankenigsberg**
```release-note
NONE
```

### CODE REVIEW

File: hack/gosec.sh [hunk 1]
```diff
@@ -7,10 +7,14 @@ mkdir -p $ARTIFACTS
...
+OUT="${ARTIFACTS}/junit-gosec.xml"
+ARGS="-sort -quiet -out=$OUT -exclude-dir=testutils -fmt=junit-xml ./..."
+if [ ! -z $GOSEC ]; then
     echo "Running subset"
     echo $GOSEC
-    gosec -include=$GOSEC -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -exclude-dir=testutils -fmt=junit-xml ./...
+    ARGS="-include=$GOSEC $ARGS"
+fi
+if ! gosec $ARGS; then
+    echo "report written to" $(echo "$OUT" | sed 's/.*_out/_out/')
```
Comment (ezrasilvera) [inline hack/gosec.sh:18]: I agree it's better to move out the gosec command and just set the args differently in the GSEC condition,
If you want to print only in the fail case, I think  you may want to add and "else" with printout of "No errors detected" or something

**Review by ezrasilvera — CHANGES_REQUESTED** [review]

**Comment by dhiller** [comment]
/override pull-kubevirt-e2e-k8s-1.17

Unrelated and flaky test.

**Pull request merged**